pkg/controller/hyperconverged/hyperconverged_controller.go

```diff
@@ -768,6 +770,67 @@ func (r *ReconcileHyperConverged) ensureKubeVirtNodeLabellerBundle(instance *hco
 	return r.client.Status().Update(context.TODO(), instance)
 }
 
+func newIMSConfigForCR(cr *hcov1alpha1.HyperConverged, namespace string) *corev1.ConfigMap {
+	labels := map[string]string{
+		"app": cr.Name,
+	}
+	return &corev1.ConfigMap{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "v2v-vmware",
+			Labels:    labels,
+			Namespace: namespace,
+		},
+		Data: map[string]string{
+			"v2v-conversion-image":              os.Getenv("CONVERSION_CONTAINER"),
+			"kubevirt-vmware-image":             os.Getenv("VMWARE_CONTAINER"),
+			"kubevirt-vmware-image-pull-policy": "IfNotPresent",
+		},
+	}
+}
+
+func (r *ReconcileHyperConverged) ensureIMSConfig(instance *hcov1alpha1.HyperConverged, logger logr.Logger, request reconcile.Request) error {
+	if os.Getenv("CONVERSION_CONTAINER") == "" {
+		return e.New("ims-conversion-container not specified")
```

Comment:
```suggestion
		return errors.New("ims-conversion-container not specified")
```

Reply:
Will change with above if you have a better suggestion.